src/core/player.rs

```diff
@@ -29,6 +31,9 @@ impl Plugin for PlayerPlugin {
     fn build(&self, app: &mut App) {
         app.add_startup_system(create_server_player_from_opts)
             .add_system_set(SystemSet::on_enter(AppState::Lobby).with_system(create_server_player));
+
+        #[cfg(feature = "gi")]
```

Comment:
You previously put it directly into a character. Is there a reason to use the system instead?

Reply:
Since now it won't compile if I put `#[cfg(feature = "gi")]` in a struct decl, that's a way to work around.